portfolio/src/main/webapp/style.css

```diff
@@ -16,14 +16,14 @@ hr.bar {
   color: #000000;
   text-align: center;
   padding: 5px 0px 15px;
-  width: 25%;
+  width: 20%;
 }
 
 .top-navigation a:hover {
   color: #f2f2f2;
 }
 
-#content {
+#content, #servlet-content {
```

Comment:
nit: separate these on different lines
```
#content,
#servlet-content {
...
}
```

https://google.github.io/styleguide/htmlcssguide.html#Selector_and_Declaration_Separation

Reply:
Noted! I changed it for the rest of the file as well :)